lib/stones-spec/language/gobstones.rb

```diff
@@ -28,6 +28,10 @@ def self.is_runtime_error?(result)
       result.include? 'Error en tiempo de ejecución'
     end
 
+    def self.is_syntax_error?(result)
+      result.include? 'Error de sintaxis' or result.include? 'Error en el programa'
```

Comment:
I think it is ok leaving it here. Adding it to array is a bit of over generalization. Unless you know you are duplicating any logic